src/Compilers/Core/Portable/PEWriter/BlobWriter.cs

```diff
@@ -628,6 +628,23 @@ internal void WriteCompressedUInt(uint val)
             }
         }
 
+        internal static int GetCompressedIntegerSize(int value)
```

Comment:
@tmat  given that they are used in two different methods I would define constants for them and add comments that point them to that portion of the spec.